crates/pbh-ctf/src/config.rs

```diff
@@ -0,0 +1,46 @@
+use std::path::Path;
+
+use reqwest::Url;
+use serde::Deserialize;
+
+pub const CONFIG_PREFIX: &str = "CTF_BOT";
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct CtfConfig {
+    /// Ethereum Signer
+    pub private_key: String,
+    /// The Wallet Nonce
+    pub wallet_nonce: u64,
```

Comment:
Do we need a wallet nonce here? Can this be inferred by the provider via a nonce manager?